app/routes.js

```diff
@@ -51,15 +51,21 @@ router.get('*/check', (req, res, next) => {
   return isDapAvailable(req, res, next);
 });
 
+router.get('*/start', (req, res, next) => {
+  req.session.data.skipStartPageValidation = skipStartPageValidation;
+  next();
+});
+
 // If a start page in the version exists,
 router.get('*/eligibility', (req, res, next) => {
   const url = req.params[0];
   const startExists = fs.existsSync('app/views' + url + '/start.html')
 
   if (startExists) {
     const data = req.session.data;
+
     // probably create an array of props and check the obj includes them all
-    if (data['caller-full-name'] && data['caller-phone-number'] && data['deceased-national-insurance'] && data['deceased-full-name']) {
+    if (data['skip-start-validation'] || (data['caller-full-name'] && data['caller-phone-number'] && data['deceased-national-insurance'] && data['deceased-full-name'])) {
```

Comment:
is there much point in saving it in session? why not just do `if (skipStartPageValidation ||`

Reply:
Since the prototype kit masks `POST` requests as `GET` requests, the only way to tell if we're "`POST`"ing is if data is available (your original check). Since we only want to skip start page validation, we need to provide a "marker" for this.

If we were ok with skipping the start page entirely, we could change it to your suggestion